app/views/notFoundError.scala.html

```diff
@@ -1,14 +1,14 @@
 @()(implicit messages: Messages, request: RequestHeader)
 
-@main(Messages("404.error.header")) {
+@main("Page not found") {
 @defining(play.core.PlayVersion.current) { version =>
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-        <h1 class="govuk-heading-l">@Messages("notFoundError.header")</h1>
-        <p class="govuk-body">@Messages("notFoundError.body1")</p>
-        <p class="govuk-body">@Messages("notFoundError.body2")</p>
-        <p class="govuk-body">@Html(Messages("notFoundError.body3"))</p>
-        <p class="govuk-body">@Html(Messages("notFoundError.body4"))</p>
+        <h1 class="govuk-heading-l">Page not found</h1>
+        <p class="govuk-body">If you typed a web address, check it is correct.</p>
+        <p class="govuk-body">If you pasted the web address, check you copied the entire address.</p>
+        <p class="govuk-body">@Html("If the web address is correct, please <a href= \"/contact\">contact<a/> us.")</p>
```

Comment:
I don't think we need the `@Html` wrapper now that the text is in the view rather than the messages file.